README.md

```diff
@@ -57,6 +57,7 @@ See also environment variables from lower layers: [`docker-openjdk`](https://git
 | SOLR_DATA_DIR | | | /opt/alfresco-search-services/data/index | solr6 only |
 | SOLR_MODEL_DIR | | | /opt/alfresco-search-services/data/model | solr6 only |
 | SOLR_CONTENT_DIR | | | /opt/alfresco-search-services/data/contentstore | solr6 only |
+| SOLR_BACKUP_DIR | solr.backup.dir | | /opt/alfresco-search-services/data/solr6Backup | solr6 ASS v2* only |
```

Comment:
You are using this variable in the generic `Dockerfile.solr6` below.

That means it is not ASS 2.x only.